### PR TITLE
code revised

### PR DESCRIPTION
**Pull request by rmakulov**
https://github.com/axibase/atsd-api-test/pull/37

### CODE REVIEW

File: src/test/java/com/axibase/tsd/api/Util.java [hunk 1]
```diff
@@ -18,6 +18,7 @@
     public static final String MAX_QUERYABLE_DATE = "9999-12-31T23:59:59.999Z";
     public static final String MIN_STORABLE_DATE = "1970-01-01T00:00:00.000Z";
     public static final String MAX_STORABLE_DATE = "2106-02-07T07:28:14.999Z";
+    public static final String NEXT_AFTER_MAX_STORABLE_DATE = "2106-02-07T07:28:15.000Z";
```
Comment (KutuzovGregory) [inline src/test/java/com/axibase/tsd/api/Util.java]: please, rewrite it with milliseconds and format

**Comment by KutuzovGregory** [comment]
ok

File: src/test/java/com/axibase/tsd/api/method/series/SeriesInsertTest.java [hunk 2]
```diff
@@ -407,15 +390,14 @@ public void testTimeRangeMaxInMSOverflow() throws Exception {
...
         final BigDecimal v = new BigDecimal("4294970895000");
 
         Series series = new Series("e-time-range-8", "m-time-range-8");
-        series.addData(new Sample(d, v));
+        series.addData(new Sample(Util.NEXT_AFTER_MAX_STORABLE_DATE, v));
 
         assertFalse("Managed to insert series with t out of range", insertSeries(series, 700));
 
-        SeriesQuery seriesQuery = new SeriesQuery(series.getEntity(), series.getMetric(), d, "2106-02-07T07:28:15.001Z");
+        SeriesQuery seriesQuery = new SeriesQuery(series.getEntity(), series.getMetric(), Util.NEXT_AFTER_MAX_STORABLE_DATE, "2106-02-07T07:28:15.001Z");
```
Comment (Sild) [inline src/test/java/com/axibase/tsd/api/method/series/SeriesInsertTest.java]: `2106-02-07T07:28:15.001Z` export to named constant please

File: src/test/java/com/axibase/tsd/api/Util.java [hunk 3]
```diff
@@ -147,4 +147,8 @@ public static String generateStringFromChar(char c, int length) {
         }
         return sb.toString();
     }
+
+    public static String addOneMS(String date) throws ParseException {
```
Comment (unrealwork) [inline src/test/java/com/axibase/tsd/api/Util.java]: May be to create  a something like `DateUtil` or `Util.Date` and replace  it there?

File: src/test/java/com/axibase/tsd/api/method/series/SeriesInsertTest.java [hunk 4]
```diff
@@ -372,16 +356,15 @@ public void testTimeRangeMaxInMSSaved() throws Exception {
...
 
         Series series = new Series("e-time-range-6", "m-time-range-6");
-        series.addData(new Sample(d, v));
+        series.addData(new Sample(MAX_STORABLE_DATE, v));
 
         Boolean success = insertSeries(series, 700);
         if (!success)
             fail("Failed to insert series");
-        SeriesQuery seriesQuery = new SeriesQuery(series.getEntity(), series.getMetric(), d, "2106-02-07T07:28:15.000Z");
+        SeriesQuery seriesQuery = new SeriesQuery(series.getEntity(), series.getMetric(), MAX_STORABLE_DATE, addOneMS(MAX_STORABLE_DATE));
```
Comment (unrealwork) [inline src/test/java/com/axibase/tsd/api/method/series/SeriesInsertTest.java]: May be make it as local constant ?

**Comment by unrealwork** [comment]
:pensive:

**Pull request merged**